content/en/docs/concepts/configuration/secret.md

```diff
@@ -106,7 +127,23 @@ system, without being directly exposed to the Pod. For example, Secrets can hold
 credentials that other parts of the system should use to interact with external
 systems on your behalf.
 
-### Using Secrets as files from a Pod
+{{< caution >}}
+Secret volume sources are validated to ensure that the specified object
+reference actually points to an object of type Secret. Therefore, a secret
+needs to be created before any Pods that depend on it.  
+
+If the secret cannot be fetched because it does not exist or
+because of a temporary lack of connection to the API server, the kubelet
+periodically retries running that Pod. The kubelet also reports an Event
+for that Pod, including details of the problem fetching the Secret.
```

Comment:
nit: Capitalization, "Event" or "event"

Reply:
We [use upper camel case for API objects](https://kubernetes.io/docs/contribute/style/style-guide/#use-upper-camel-case-for-api-objects), and events are objects in the Kubernetes API.

Reply:
✔️